README.md

```diff
@@ -96,6 +96,7 @@ is not needed to install packages with these frameworks:
 | Porto        | `porto-container`
 | RadPHP       | `radphp-bundle`
 | REDAXO       | `redaxo-addon`
+| REDAXO V5.*  | `redaxo5-addon`
```

Comment:
You may need to add `redaxo5-bestyle-plugin`?

Reply:
I also added the missing redaxo-bestyle-plugin